www/feeds/atom/search.php

```diff
@@ -5,9 +5,11 @@
 
 try{
 	$query = HttpInput::Str(GET, 'query') ?? '';
+	$startPage = HttpInput::Str(GET, 'page') ?? 1;
```

Comment:
This and count should be `HttpInput::Int()` not Str, in all of the search pages in this PR

Reply:
Good catch, fixed.